qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -207,5 +207,13 @@ def write_local_config(self):
             export CEPHFS_MOUNT_OPTIONS="-o name=admin,secret={self.get_admin_key()}"
             ''')
 
-       self.mount_a.client_remote.write_file(join(self.xfstests_repo_path, 'local.config'),
-                                             xfstests_config_contents, sudo=True)
+        config_path = join(self.xfstests_repo_path, 'local.config')
+        self.mount_a.client_remote.write_file(config_path,
```

Comment:
How is this making debugging easier? You could just dump `xfstests_config_contents` rather than writing and reading it back again.

Reply:
I was just being cautious considering that in future someone might change this code in that case `xfstests_config_contents` and actually file contents might be different. In such a case developer would be misled during debugging `xfstests_dev.py`

Reply:
That code will be vetted before merge anyway.

Reply:
@rishabh-d-dave in the latest update, this still write to and reads from the file. Did you miss removing that?

Reply:
It's changed now.